ui/src/pages/server-types/server-types.tsx

```diff
@@ -210,9 +221,7 @@ export const ServerTypes = (): React.ReactElement => {
           .
         </p>
       </div>
-      {error ? (
-        <div>An error occurred: {(error as Error).message}</div>
-      ) : isLoading ? (
```

Comment:
Removed and replaced with current pattern or displaying overall errors at app level